wee_slack.py

```diff
@@ -2694,6 +2694,8 @@ def modify_buffer_line(buffer, new_line, timestamp, time_id):
         # hold the structure of a line and of line data
         struct_hdata_line = w.hdata_get('line')
         struct_hdata_line_data = w.hdata_get('line_data')
+        # keep track of the number of lines with the matching time and id
+        number_of_matching_lines = 0
 
         while line_pointer:
             # get a pointer to the data in line_pointer via layout of struct_hdata_line
```

Comment:
Good idea. I added some error handling so the modification will be aborted if it happens, instead of something unexpected happening. Please tell me if you want to log differently, or have a better error message. Possibly log to a more visible place than debug?